site/en/codelabs/silabs-openthread-hardware/index.lab.md

```diff
@@ -0,0 +1,438 @@
+| 3     | `dataset commit active`                              | Commit new dataset to the Active Operational Dataset         | Done              |
+| 4     | `ifconfig up`                                        | Enable Thread interface                                      | Done              |
+| 5     | `thread start`                                       | Enable and attach Thread protocol operation.                 | Done              |
+|       |                                                      | Wait 20 seconds while the device joins and configures itself. |                   |
+| 6     | `state`                                              | View network configuration.                                  | child<br/>Done    |
+
+> Note: Because of the self configuring nature of a Thread network and these being Full Thread devices, either or both the FTDs may eventually become  routers. This can be found out using the `state` command.
+
+### Communication between Thread devices
+
+We will use the `ping` command to check if devices can communicate with each other. To use the ping command we need the IPv6 addresses of the devices. These can be obtained using the `ipaddr` command.
+
+```console
+> ipaddr																	# IPv6 addresses of the OTBR
+fd5c:c6b:3a17:40b9:0:ff:fe00:fc00					# Leader Anycast Locator (ALOC)
+fd5c:c6b:3a17:40b9:0:ff:fe00:1800					# Routing Locator (RLOC)
+fd5c:c6b:3a17:40b9:84e2:bae8:bd5b:fa03		# Mesh-Local EID (ML-EID)
+fe80:0:0:0:c449:ca4a:101f:5d16						# Link-Local Address (LLA)
+```
+
+From both FTDs ping the OTBR using the OTBR's RLOC address.
+
+```console
+ping fd5c:c6b:3a17:40b9:0:ff:fe00:1800
+Done
+> 
+> 16 bytes from fd5c:c6b:3a17:40b9:0:ff:fe00:1800: icmp_seq=3 hlim=64 time=30ms
+16 bytes from fd5c:c6b:3a17:40b9:0:ff:fe00:1800: icmp_seq=3 hlim=64 time=52ms
+```
+
+ The response indicates the payload received and that the communication is successful. Repeat the process to ping FTDs from the OTBR.
+
+**You've created a Thread network!**
+
+ You now know:
+
+* How to create an OpenThread project using Silicon Labs Simplicity Studio IDE.
+* How to build and flash OpenThread CLI binaries to Silicon Labs radio boards.
+* How to setup a Raspberry Pi 3B+ or greater as an OpenThread Border Router (OTBR) using Docker.
+* How to create a Thread network on the OTBR.
+* Out of band commissioning of devices onto a Thread network.
+* How to verify Thread communication between the nodes using the ping command.
+
+## Further reading
+
+Check out  [openthread.io](https://openthread.io) and [GitHub](https://github.com/openthread) for a variety of OpenThread resources, including:
+
+*  [Supported Platforms](https://openthread.io/platforms/)
+    — discover all the platforms that support OpenThread
+    
+*  [Build OpenThread](https://openthread.io/guides/build/)
```

Comment:
For files that are in this repo, we're using relative links now

../../guides/build/index.md